extensions/dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -120,6 +131,49 @@ function _getDisplaySetsFromSeries(
   return [displaySet];
 }
 
+function _isRehydratable(displaySet, MeasurementService) {
```

Comment:
Shift this to a separate file. Have it take in `mappings` instead of `MeasurementService`?

Reply:
JSDoc